src/newItinerary/index.tsx

```diff
@@ -1,7 +1,8 @@
 export { Itinerary, ItineraryProps } from './Itinerary'
-export { Addon } from './Addon'
-export { HiddenStops } from './HiddenStops'
-export { Address } from './Address'
-export { HiddenStop } from './HiddenStop'
-export { Connection } from './Connection'
+export { Addon, AddonProps } from './Addon'
+export { HiddenStops, HiddenStopsProps } from './HiddenStops'
+export { Address, AddressProps } from './Address'
+export { HiddenStop, HiddenStopProps } from './HiddenStop'
+export { Connection, ConnectionProps } from './Connection'
 export { Lines } from './Lines'
+export { Proximity, ProximityDisplay, ProximityProps, Distances } from './Proximity'
```

Comment:
not even nit since it's not really connected to this PR but I'd feel better if all the sub components of `newItinerary` had their own folder